changelog/Microsoft.DirectoryServices.json

```
@@ -7383,6 +7383,152 @@
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Method",
+          "ChangedApiName": "identityProviderBase",
+          "ChangeType": "Addition",
+          "Description": "To support listing all identity provider configurations in an Azure AD and Azure AD B2C tenant, introduced [list](https://docs.microsoft.com/en-us/graph/api/identityproviderbase-list?view=graph-rest-1.0) operation.",
+          "Target": "identityProviderBase"
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Method",
+          "ChangedApiName": "identityProviderBase",
+          "ChangeType": "Addition",
+          "Description": "To support getting an identity provider configuration in an Azure AD and Azure AD B2C tenant, introduced [get](https://docs.microsoft.com/en-us/graph/api/identityproviderbase-get?view=graph-rest-1.0) operation.",
+          "Target": "identityProviderBase"
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Method",
+          "ChangedApiName": "identityProviderBase",
+          "ChangeType": "Addition",
+          "Description": "To support updating an identity provider in an Azure AD and Azure AD B2C tenant, introduced [update](https://docs.microsoft.com/en-us/graph/api/identityproviderbase-update?view=graph-rest-1.0) operation.",
+          "Target": "identityProviderBase"
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Method",
+          "ChangedApiName": "identityProviderBase",
+          "ChangeType": "Addition",
+          "Description": "To support deleting an identity provider in an Azure AD and Azure AD B2C tenant, introduced [delete](https://docs.microsoft.com/en-us/graph/api/identityproviderbase-delete?view=graph-rest-1.0) operation.",
+          "Target": "identityProviderBase"
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Method",
+          "ChangedApiName": "identityProviderBase",
+          "ChangeType": "Addition",
+          "Description": "To support listing available provider types of identity provider in an Azure AD and Azure AD B2C tenant, introduced [availableProviderType](https://docs.microsoft.com/en-us/graph/api/identityprovider-list-availableprovidertypes?view=graph-rest-1.0) operation.",
+          "Target": "identityProviderBase"
+        },
+        {
+          "Id": "224526a5-09a4-4895-9b8b-40e8984a4111",
+          "ApiChange": "Resource",
+          "ChangedApiName": "identityProvider",
+          "ChangeType": "Change",
+          "Description": "Deprecated the [identityProvider](https://docs.microsoft.com/en-us/graph/api/resources/identityprovider?view=graph-rest-1.0) entity to replace it with new [identity provider API](https://docs.microsoft.com/en-us/graph/api/resources/identityproviderbase?view=graph-rest-1.0).",
```

- Suggestion applies to other entries
- If all its methods are deprecated, just say "... and related operations and methods" instead and remove the corresponding entries

```suggestion
          "Description": "Deprecated the [identityProvider](https://docs.microsoft.com/en-us/graph/api/resources/identityprovider?view=graph-rest-1.0) resource to replace it with new [identity provider API](https://docs.microsoft.com/en-us/graph/api/resources/identityproviderbase?view=graph-rest-1.0).",
```